src/Common/WikiArticles.fs

```diff
@@ -15,3 +15,19 @@ type VerbArticle = VerbArticle of Article
 type VerbArticleWithImperative = VerbArticleWithImperative of VerbArticle
 type VerbArticleWithParticiple = VerbArticleWithParticiple of VerbArticle
 type VerbArticleWithConjugation = VerbArticleWithConjugation of VerbArticle
+
+type AdjectiveDeclension = {
+    Singular: string
+    Plural: string
+}
+
+type AdjectiveComparison = {
+    Positive: string
+    Comparatives: seq<string>
+}
+
+type Adjective = {
+    CanonicalForm: string
+    Declension: AdjectiveDeclension option
```

Comment:
why is it optional? are there cases when we have no declension / comparison?

Reply:
Yes. As for comparison, a class of adjectives (called [absolute](https://en.wikipedia.org/wiki/Comparison_(grammar)#Absolute_adjectives) or [vztahové](https://prirucka.ujc.cas.cz/?slovo=krasny#nadpis2_1)) cannot really have degrees, e.g. [pražský](https://cs.wiktionary.org/wiki/pra%C5%BEsk%C3%BD#p%C5%99%C3%ADdavn%C3%A9_jm%C3%A9no). 

Another thing, both sections can be just not filled in wiki, we cannot rely on that regretfully.